hbase/Dockerfile

```diff
@@ -66,5 +84,37 @@ COPY shared/log4shell_scanner /bin/log4shell_scanner
 RUN /bin/log4shell_scanner s /stackable/hbase-${PRODUCT}
 # ===
 
+
+# Final Image
+FROM stackable/image/java-base
+
+ARG PRODUCT
+ARG RELEASE
+
+LABEL name="Apache HBase" \
+      maintainer="info@stackable.tech" \
+      vendor="Stackable GmbH" \
+      version="${PRODUCT}" \
+      release="${RELEASE}" \
+      summary="The Stackable image for Apache HBase." \
+      description="This image is deployed by the Stackable Operator for Apache HBase."
+
+# https://github.com/hadolint/hadolint/wiki/DL4006
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# krb5-workstation for Kerberos support
+RUN microdnf update && \
+    microdnf install \
+    krb5-workstation && \
+    microdnf clean all && \
+    rm -rf /var/cache/yum
+
+COPY --chown=stackable:stackable --from=builder /stackable/hbase-${PRODUCT} /stackable/hbase-${PRODUCT}/
+COPY --chown=stackable:stackable --from=builder /stackable/jmx /stackable/jmx/
```

Comment:
I think the softlinks for jmx-exporter will not be copied here removing the image backwards compability.
It is in here though https://repo.stackable.tech/#browse/browse:docker:v2%2Fstackable-experimental%2Fhbase%2Ftags%2F2.4.17-stackable23.11.0-compression-fix. 
Was this build with the same flow?